test/testcafe/framework/page-objects/EnrollProfileViewPageObject.js

```diff
@@ -1,5 +1,7 @@
 import BasePageObject from './BasePageObject';
 
+const requirementsSelector = '[data-se="password-authenticator--rules"]';
```

Comment:
feel free to inline this

Reply:
I ultimately got rid of this and used the `getByText` function instead based on your other feedback. 😄